Doc/glossary.rst

```diff
@@ -221,6 +221,14 @@ Glossary
       statement by defining :meth:`__enter__` and :meth:`__exit__` methods.
       See :pep:`343`.
 
+    context variable
+      A variable which can have different values depending on it's context.
```

Comment:
This would be `its context` instead of `it's context`.

Reply:
Done.